docs/source/global_settings.rst

```diff
@@ -0,0 +1,20 @@
+Global Settings
+================
+
+The global_settings.py module is the home to the global variable GAMEBENCH_CONFIG.
+This dictionary holds the API endpoint information that the library uses.
+It also contains the username and password for the GameBench account you are using,
+and the company ID.
+
+.. code-block:: python
```

Comment:
This is a sneaky syntax gotcha! The option needs to align with the directive name or it won't render correctly in ReadTheDocs.

```suggestion
.. code-block:: python
   :linenos:
```

Reply:
Fixed and added the :linenos: option!